### ISSUE TITLE
[Enhancement] Match platforms to Meta store (eg Quest 1/2/3/etc)

### ISSUE THREAD

**Issue by DanDoes** [Jeshibu/Playnite-OculusLibrary#11]
Version: 2.6.1

Not sure if this is possible, but ideally I'd be able to see the same Platforms in Playnite that Meta lists in their store, with per game entry:
![image](https://github.com/user-attachments/assets/ef1e5472-f115-4db0-9b23-f29269e8db4b)

---

Similar to how Playnite's Xbox plugin behaves:
![image](https://github.com/user-attachments/assets/7197ebfd-dc3d-4f05-9770-418fd8e51a48)

![image](https://github.com/user-attachments/assets/2d3b6ce2-c2bc-42bf-ae81-fbb095d7ef10)

---

Why?
* To know what's available, where.
* I only have a Quest 1, so I can't play _all_ Oculus games - I'd like to be able to filter to the ones I can actually play.

**Comment by DanDoes** [comment]
From [this suggestion](https://github.com/Jeshibu/Playnite-OculusLibrary/issues/10#issuecomment-2351112927), it sounds like this may be possible via metadata import from "Official Store", but I tried that and they all remained "Meta Quest" 🤔

**Comment by Jeshibu** [comment]
Right click game > Edit, then this:
![image](https://github.com/user-attachments/assets/f2f999e9-7d1d-4f53-a97e-e3bbd1df2067)

Works for me:
![image](https://github.com/user-attachments/assets/002de19f-4c4d-4ec6-a49e-157d6d49465f)

If you want to do this for every game you've imported:

1. Filter to your Oculus/Meta games: https://api.playnite.link/docs/master/manual/features/filtersAndFiltersPresets.html
2. Update metadata (make sure to uncheck the "Only missing metadata" checkbox and select "Official Store" for the Platform field): https://api.playnite.link/docs/master/manual/library/games/metadata.html#updating-game-metadata

**Comment by DanDoes** [comment]
Yeah I tried both of those, here's what that looks like for me, no visible errors or anything:
https://www.imghippo.com/i/37ffP1726354604.gif
😕
